docs/sphinx/source/installation.rst

```diff
@@ -19,9 +25,21 @@ If you are on a PC you should also copy the `Jaloxa radwinexe-5.0.a.8-win64.zip
 
 **Note: bifacial_radiance is not endorsed by or officially connected with the Radiance software package or its development team.**
   
+
+Prerequisite: PYTHON:
+~~~~~~~~~~~~~~~~~~~~~~
+You will need python installed to run bifacial_radiance. We suggest using the latest release of `Anaconda with Python 3.7 <https://www.anaconda.com/distribution/>`_ (Python 2.7 is still supported but in the process of being deprecated). Anaconda will install ``Spyder`` to work with the python scripts, and also it will install ``Jupyter``, which is the tool we use for our `tutorial trainings <https://github.com/NREL/bifacial_radiance/tree/master/docs/tutorials>`_
+
+
 STEP 1
 ~~~~~~
-Install and Import bifacial_radiance
+
+The simplest option is to open a command prompt and run::
+
+        pip install bifacial_radiance
+        
+       
+An alternative which is shown in the Video Instructions, if you want to install bifacial_radiance in a local folder of your choosing and/or be able to modify the internal code to suit your needs, you can do the following:
 
 * clone the bifacial_radiance repo to your local directory or download and unzip the .zip file
 * navigate to the \bifacial_radiance directory using anaconda command line
```

Comment:
The line below on `deploy in an Anaconda 2.7` probably needs to be updated to indicate a more recent version.

Reply:
Updated to Anaconda 2019.10